src/core_plugins/vega/public/data_model/ems_file_parser.js

```diff
@@ -35,7 +36,13 @@ export class EmsFileParser {
    */
   parseUrl(obj, url) {
     if (typeof url.name !== 'string') {
-      throw new Error(`data.url with {"%type%": "emsfile"} is missing the "name" of the file`);
+      throw new Error(i18n.translate('vega.emsFileParser.missingNameOfFileErrorMessage', {
+        defaultMessage: '{dataUrlParam} with {dataUrlParamValue} is missing the "name" of the file',
```

Comment:
I think `"name"` also needs to be pulled out to a parameter, since it's a JavaScript property name, that should not be translated.